ciphers/rail_fence_cipher.py

```diff
@@ -1,44 +1,29 @@
-string = input("Enter string to be encrypted:")
-print("Enter key for encryption (works best for small key values) : ", end = "")
+input_string = input()
 key = int(input())
 
-while key >= len(string):
-	print("Please enter a smaller value: ", end = "")
-	key = int(input())
-	
-def rail_fence_encrypt(string, key):
-	a = [[None for i in range(len(string))] for j in range(key)]
+string_list = [["" for _ in range(len(input_string))]for _ in range(key)]
+
+def rail_fence_encryption(string, key):
```

Comment:
```suggestion
def rail_fence_encryption(msg: str, key: str) -> str:
```

`string` is a Python Standard Library module do it should not be used as a variable name.

https://docs.python.org/3/library/string.html